qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -41,9 +41,9 @@ def run_test(self):
     def add_options(self, parser):
         pass
 
-    def setup_chain(self):
+    def setup_chain(self,bitcoinConfDict=None, wallets=None):
```

Comment:
these new parameters need some brief introduction, here or in another suitable place (docstring)